core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/view/ClusterApiViewToClusterViewV4ResponseConverter.java

```diff
@@ -49,6 +49,7 @@ protected ClusterViewV4Response createTarget() {
     private void addSharedServiceResponse(ClusterApiView cluster, ClusterViewV4Response clusterResponse) {
         SharedServiceV4Response sharedServiceResponse = new SharedServiceV4Response();
         if (cluster.getStack().getDatalakeId() != null) {
+            sharedServiceResponse.setResourceCrn(cluster.getStack().getResourceCrn());
```

Comment:
just be aware we are sending back the stack crn and no relation with the sdx cluster crn, can be confusing as you can use sdx crn for querying

Reply:
We need the sdx crn

Reply:
reworked